scrapers/scrape_be.sh

```diff
@@ -1,39 +1,91 @@
+                <col width="10%"></col>
+                <col width="10%"></col>
+                <col width="10%"></col>
+                <col width="10%"></col>
+                <col width="10%"></col>
+                <col width="10%"></col>
                 </colgroup>
         <thead>
         <tr>
-                <th id="th_top_5147_1A">Fälle</th>
-                        <th id="th_top_5147_2A">Todesfälle</th>
+                <th id="th_top_5147_1A"><strong>Datum</strong></th>
+                        <th id="th_top_5147_2A"><strong>Fälle</strong><br />
+positiv</th>
+                        <th id="th_top_5147_3A">Im<br />
+<strong>Spital</strong><br />
+gesamt</th>
+                        <th id="th_top_5147_4A">Davon<br />
+normale<br />
+<strong>Betten-<br />
+ station</strong></th>
+                        <th id="th_top_5147_5A">Davon<br />
+<strong>Intensiv-<br />
+ station</strong><br />
+gesamt</th>
+                        <th id="th_top_5147_6A">Davon<br />
+Intensiv-<br />
+pflege<br />
+<strong>beatmet</strong></th>
+                        <th id="th_top_5147_7A"><strong>Todes-<br />
+ fälle</strong></th>
                         </tr>
         </thead>
         <tbody>
         <tr>
-                <td headers="th_top_5147_1A"><strong>624</strong></td>
-                        <td headers="th_top_5147_2A">6</td>
+                <td headers="th_top_5147_1A">30.03.2020</td>
+                        <td headers="th_top_5147_2A">826</td>
+                        <td headers="th_top_5147_3A">112</td>
+                        <td headers="th_top_5147_4A">91</td>
+                        <td headers="th_top_5147_5A">21</td>
+                        <td headers="th_top_5147_6A">17</td>
+                        <td headers="th_top_5147_7A">13</td>
                         </tr>
         </tbody>
     </table>
-</div></div>
-<a name="middlePar_textbild_4bad" style="visibility:hidden"></a><div class="textBild floatingComponent section"><p>(Stand: 25. März 2020)</p>
 """
 
-print('Date and time:', sc.find(r'Stand: (.+)\)', d))
-# This isn't really the best method, but it works for now.
-print('Confirmed cases:', sc.find(r'<td .*<strong>([0-9]+)<', d))
-print('Deaths:', sc.find(r'<td[^<>]*>([0-9]+)</td>', d))
+m = re.search(r'<table .*? summary="Laufend aktualisierte Zahlen zu den Corona-Erkrankungen im Kanton Bern">(.*?)<\/table>', d, flags=re.MULTILINE | re.DOTALL)
+assert m, "Can't find table"
+
+d = m[1].replace('\n', '')
+
+header = sc.find("<thead>\s*<tr>\s*(<th .*><strong>Datum</strong></th>\s*<th .*><strong>Fälle</strong><br />positiv</th>\s*<th .*>Im<br /><strong>Spital</strong><br />gesamt</th>\s*<th .*>Davon<br />normale<br /><strong>Betten-<br /> station</strong></th>\s*<th .*>Davon<br /><strong>Intensiv-<br /> station</strong><br />gesamt</th>\s*<th .*>Davon<br />Intensiv-<br />pflege<br /><strong>beatmet</strong></th>\s*<th .*><strong>Todes-<br /> fälle</strong></th>)\s*</tr>\s*</thead>", d)
```

Comment:
Use non-greedy matching for the `<th>` tag, like this: `<th.*?>`

Reply:
Done!